modules/nlu-testing/src/bot-templates/bp-nlu-synonyms-testing/intents/travel.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "travel",
+  "utterances": {
+    "en": [
+      "Where is the center ? ",
+      "What can I visit in Paris ? ",
+      "How can I go to the Louvre ? ",
+      "I'm searching a romantic place to spend the evening",
+      "Which neighborhood are worth visiting ?",
+      "Is it expensive to travel in paris ?",
+      "Which is the fastest way to go to the airport ?",
+      "Is the city adapted to bring animals ?"
+    ]
+  },
+  "slots": [],
+  "contexts": ["splitable"]
```

Comment:
Not sure I understand the purpose of `splitable` just looking at data and the tests, could you add a summary of the edge-cases in the readme of this bot?

Reply:
Splitable is the baseline to ensure the bot can perform on two really different intents